### PR TITLE
{tools}[GCCcore/11.2.0] bzip2 v1.0.8

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/b/bzip2/bzip2-1.0.8-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ techniques (the PPM family of statistical compressors), whilst being around
+ twice as fast at compression and six times faster at decompression.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://sourceware.org/pub/%(name)s/']
+sources = [SOURCE_TAR_GZ]
+patches = ['bzip2-1.0.6-pkgconfig.patch']
```
Comment (branfosj) [inline easybuild/easyconfigs/b/bzip2/bzip2-1.0.8-GCCcore-11.2.0.eb:17]: We should fix the patch. See https://github.com/easybuilders/easybuild-easyconfigs/issues/13513

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/bzip2/bzip2-1.0.8-GCCcore-11.2.0.eb (hunk 1 above)
Comment (SebastianAchilles) [reply easybuild/easyconfigs/b/bzip2/bzip2-1.0.8-GCCcore-11.2.0.eb:17]: Thank you! I wasn't aware of this issue. There is no open PR yet for the updated patch, right?

**Comment by SebastianAchilles** [comment]
Closing because this is part of https://github.com/easybuilders/easybuild-easyconfigs/pull/13581 with the updated patch.

**Pull request closed without merging**